src/modules/images/Preview.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react';
+import {View, TouchableOpacity, Image, ActivityIndicator} from 'react-native';
+
+import {widthPercentageToDP as wp, heightPercentageToDP as hp} from 'react-native-responsive-screen';
+import {useDispatch, useSelector} from 'react-redux';
+
+import IconButton from '../../shared/ui/IconButton';
+import ImagePropertiesModal from '../images/ImagePropertiesModal';
+import {NotebookPages} from '../notebook-panel/notebook.constants';
+import {spotReducers} from '../spots/spot.constants';
+import useSpotsHook from '../spots/useSpots';
+import styles from './images.styles';
+
+
+const Preview = (props, item,
+                 imageKey,) => {
+  const [useSpots] = useSpotsHook();
+  const spots = useSelector(state => state.spot.spots);
+  const [imageNoteModal, setImageNoteModal] = useState(false);
+  const dispatch = useDispatch();
+  let noteModal = (
+    <View style={styles.modalPosition}>
+      <ImagePropertiesModal
+        close={() => closeModal()} // Saves and closes modal
+        cancel={() => closeModal()} // Closes without saving
+      >
+        Image Info
+      </ImagePropertiesModal>
+    </View>);
```

Comment:
); should be on next line so it lines up with the same line where the opening parenthesis